docs/howto/openstack/nova/server-group.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Server groups provide a mechanism to group servers according to a certain policy.
+---
+# Using server groups
+
+In {{brand}}, you can use server groups to control the scheduling of a group of servers.
+
+## Prerequisites
+
+In order to use server groups you must use the OpenStack CLI. Make sure you have [enabled it](/howto/getting-started/enable-openstack-cli).
+
+## Policies
+
+A server group can have one of four different policies: affinity, soft-affinity, anti-affinity or soft-anti-affinity.
+
+### Affinity
+
+A server group with the policy of `affinity` will make sure that all the servers in that group are **always** placed on the same physical compute node.
+
+### Soft affinity
+
+A policy of `soft-affinity` will **try to** make sure that all the servers in that group are placed on the same physical compute node, but ultimately will allow it if otherwise not possible.
+
+### Anti-affinity
+
+A server group with the policy of `anti-affinity` will make sure that the servers in that group are **never** placed on the same physical compute node.
+
+### Soft anti-affinity
+
+A policy of `soft-anti-affinity` will **try to** make sure that all the servers in that group are not placed on the same physical compute node, but ultimately will allow it if otherwise not possible.
+
+## Creating server groups
+
+To create a server group, use the following command:
+
+```bash
+openstack server group create \
+  --policy [affinity|soft-affinity|anti-affinity|soft-anti-affinity] \
+  <server_group_name>
+```
+
+With an `anti-affinity` or `soft-anti-affinity` policy, you may also configure how many servers you want to allow on the same physical compute node. To do this, use the option `--rule max_server_per_host=<number>`, where `<number>` is the amount of servers to allow on the same physical compute node.
+
+
+## Creating servers using server groups
+
+To apply a server group policy, you must specify the group when creating a server, as a *scheduling hint.* To do that, use the `--hint` parameter in the following command:
+
+```bash
+openstack server create --hint group=<server_group_id> [...] <server_name>
+```
+
+If you subsequently launch more servers referencing the same server group, {{brand}} concentrates or distributes them according to the server group's policy.
+
+## Troubleshooting common issues
+
+If you keep creating servers within a server group with a policy of `anti-affinity`, you will eventually exceed the total amount of physical compute nodes in the region. The command will still succeed but the server will fail in scheduling, with the following "fault" message: _No valid host was found. There are not enough hosts available._
```

Comment:
Instead of "but the server will fail in scheduling" I would use "but server scheduling will fail".

Reply:
I used some slightly clearer wording (I hope).